### ISSUE TITLE
Pandas 1.2.1 breaks pandarallel

### ISSUE THREAD

**Issue by willprice** [nalepae/pandarallel#133]
```
pd.DataFrame({
    'x': [1,1,2,2],
    'y': [1,2,3,4]
}).groupby('x').parallel_apply(lambda g: sum(g['y']))
```
Causes the following error

```
RemoteTraceback                           Traceback (most recent call last)
RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/home/will/.miniconda3/envs/proj/lib/python3.8/multiprocessing/pool.py", line 125, in worker
    result = (True, func(*args, **kwds))
  File "/home/will/.miniconda3/envs/proj/lib/python3.8/multiprocessing/pool.py", line 48, in mapstar
    return list(map(*args))
  File "/home/will/.miniconda3/envs/proj/lib/python3.8/site-packages/pandarallel/pandarallel.py", line 64, in global_worker
    return _func(x)
  File "/home/will/.miniconda3/envs/proj/lib/python3.8/site-packages/pandarallel/pandarallel.py", line 108, in wrapper
    result = function(
  File "/home/will/.miniconda3/envs/proj/lib/python3.8/site-packages/pandarallel/data_types/dataframe_groupby.py", line 36, in worker
    mutated.append(not pd.core.groupby.ops._is_indexed_like(res, df.axes))
TypeError: _is_indexed_like() missing 1 required positional argument: 'axis'
"""

The above exception was the direct cause of the following exception:

TypeError                                 Traceback (most recent call last)
<ipython-input-30-b3fb20426bf2> in <module>
----> 1 pd.DataFrame({
      2     'x': [1,1,2,2],
      3     'y': [1,2,3,4]
      4 }).groupby('x').parallel_apply(lambda g: sum(g['y']))

~/.miniconda3/envs/proj/lib/python3.8/site-packages/pandarallel/pandarallel.py in closure(data, func, *args, **kwargs)
    451             pool.close()
    452 
--> 453             results = get_workers_result(
    454                 use_memory_fs,
    455                 nb_workers,

~/.miniconda3/envs/proj/lib/python3.8/site-packages/pandarallel/pandarallel.py in get_workers_result(use_memory_fs, nb_workers, show_progress_bar, nb_columns, queue, chunk_lengths, input_files, output_files, map_result)
    394                 progress_bars.update(progresses)
    395 
--> 396     results = map_result.get()
    397 
    398     return (

~/.miniconda3/envs/proj/lib/python3.8/multiprocessing/pool.py in get(self, timeout)
    769             return self._value
    770         else:
--> 771             raise self._value
    772 
    773     def _set(self, i, obj):

TypeError: _is_indexed_like() missing 1 required positional argument: 'axis'
```

Looks like the API for `pd.core.groupby.ops._is_indexed_like` has now changed and pandarallel needs updating to be compatible.

This doesn't happen on pandas 1.1.5

**Comment by e-nalepa** [comment]
This PR solves it:
https://github.com/nalepae/pandarallel/pull/132

(Should be merged soon)

**Comment by willprice** [comment]
Super, thanks @e-nalepa, I didn't spot that as i just looked through issues rather than open PRs for dups.

**Comment by nalepae** [comment]
No pb !

Le jeu. 4 févr. 2021 à 16:53, Will Price <notifications@github.com> a
écrit :

**Comment by nalepae** [comment]
Fixed in `pandarallel v1.5.2`.
